Code/Framework/AzCore/Tests/Math/MathUtilsTests.cpp

```diff
@@ -58,4 +58,55 @@ namespace UnitTest
         EXPECT_NEAR(0.6, AZ::LerpInverse(1.0, 1.0 + 5.0 * epsilonD, 1.0 + 3.0 * epsilonD), epsilonD);
         EXPECT_NEAR(1.0, AZ::LerpInverse(1.0, 1.0 + 5.0 * epsilonD, 1.0 + 5.0 * epsilonD), epsilonD);
     }
+
+    class RoundUpToMultipleTestsFixture
```

Comment:
suggestion: Is it worth also having a sanity check on `DivideAndRoundUp` separately? It does not have to be as exhaustive and I realize it is being tests as `RoundUpToMultiple` is implemented in-terms of it, but a separate sanity check could be useful to ensure it gives the expected output

Reply:
+1